sdk/pinocchio/src/sysvars/tests/slot_hashes_tests.rs

```diff
@@ -0,0 +1,855 @@
+#[cfg(test)]
+use super::*;
+mod tests {
+    use super::*;
+    use core::mem::{align_of, size_of};
+    extern crate std;
```

Comment:
This probably shows my lack of knowledge.
But I thought that since Rust 2018 `extern crate` is no longer in use, replaced by `use` imports.
Is this case somehow special?
If so, maybe leave a comment explaining why it is done this way here?